sci-astronomy/stellarium/stellarium-24.3.recipe

```diff
@@ -52,6 +53,8 @@ BUILD_REQUIRES="
 	devel:libQt6SerialPort$secondaryArchSuffix
 	devel:libQt6Widgets$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
+	qxlsx_devel$secondaryArchSuffix
+	qxlsx_qt6$secondaryArchSuffix
```

Comment:
Ideally, this shouldn't be required, but qxlsx_devel is missing a requirement on qxlsx_qt6, it seems ...

Reply:
I fixed this in 82168a5, so this line can be removed after rebasing this PR.